uat/custom-components/src/main/resources/recipes/logGenerator.yaml

```diff
@@ -0,0 +1,18 @@
+#
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+
+---
+RecipeFormatVersion: '2020-01-25'
+ComponentName: logGenerator
+ComponentVersion: '0.0.0'
+ComponentDescription: Component to generate log files with random log messages.
+ComponentPublisher: AWS
+Manifests:
+  - Artifacts:
+      - URI: classpath:/local-store/artifacts/custom-components.jar
+    Lifecycle:
+      Run: >-
+        java -Dlog.level=INFO -DcomponentName="logGenerator" -jar {artifacts:path}/custom-components.jar 
+        "{configuration:/FileSize}" "{configuration:/LogSpeed}"
```

Comment:
```suggestion
        "{configuration:/FileSize}" "configuration:/FileSizeUnit" "{configuration:/LogWriteFrequencySeconds}"
```

Reply:
This is just temporary for testing if the current toy component can receive parameters. They should be well-defined once the custom component is developed.